tools/harness-automation/cases/ed_9_2_18.py

```diff
@@ -33,7 +33,7 @@
 from autothreadharness.harness_case import HarnessCase
 
 class ED_9_2_18(HarnessCase):
```

Comment:
class name should be MED_9_2_18  and file name change to med_9_2_18.py according to the device category change